src/coh3/coh3-data.ts

```diff
@@ -281,6 +281,23 @@ const matchTypesAsObject: Record<number, { id: number; name: string; localizedNa
   },
 };
 
+const maps: { [key: string]: { name: string; url: string } } = {
+  twin_beach_2p_mkii: { name: "Twin Beaches", url: "https://i.imgur.com/LkyRTjr.png" },
+  desert_village_2p_mkiii: { name: "Road to Tunis", url: "https://i.imgur.com/CXmRwe2.png" },
+  cliff_crossing_2p: { name: "Taranto Coastline", url: "https://i.imgur.com/gk6BZqH.png" },
+  rails_and_sand_4p: { name: "Campbell's Convoy", url: "https://i.imgur.com/6lnVXNv.png" },
```

Comment:
Would it be possible to download these images ? And add them to this folder? 
https://github.com/cohstats/coh3-stats/tree/master/public

You can keep this data type. It's pretty good 👍  the url would than be `/maps/name_of_file.png` assuming the file is in folder /public/maps/name_of_file

It would be good to have this under the control, in case they would be deleted from imgur, no-one will delete them from our repo :)